docs/content/en/docs/references/cli/_index.md

```diff
@@ -413,6 +415,7 @@ Options:
   -n, --namespace='': Run deployments in the specified namespace
       --no-prune=false: Skip removing images and containers built by Skaffold
       --no-prune-children=false: Skip removing layers reused by Skaffold
+      --platform=[]: The platform to target for the build artifacts
```

Comment:
Should we validate platforms? e.g. `arm64, amd` etc.

Reply:
we validate it here: https://github.com/GoogleContainerTools/skaffold/blob/0fd9e46a7928dcdc6c3a139b5612fb9331bfc889/pkg/skaffold/platform/platform.go#L88-L91